docs/metrics/accelerate.md

```diff
@@ -9,80 +9,86 @@ Pulse calculates the four key Accelerate metrics using strictly **changes**, **d
+
+```text
+median(incident resolved timestamp - incident created timestamp)
+```
 
-Median of `incidents` time resolved - `incidents` time created.
+Pulse uses the maximum value over a period of time to display aggregate median times to recover.
 
-When we aggregate Median time to recover we use the maximum of a period of time.
+Pulse determines your performance level for this metric as follows:
 
-| Performance level[^1] | Value                      |
+| Performance level[^1] | Median time to recover     |
 | --------------------- | -------------------------- |
-| Low                   | Between 1 week and 1 month |
-| Medium                | Less than 1 day            |
-| High                  | Less than 1 day            |
 | Elite                 | Less than 1 hour           |
+| High                  | Less than 1 day            |
+| Medium                | Less than 1 day            |
+| Low                   | Between 1 week and 1 month |
 
 ## Change failure rate
 
-Percentage of deployments causing a failure in production (e.g., service impairment or unplanned outage) and that subsequently require remediation.
-
-Number of `deployments` that caused `incidents` / Number of `deployments`.
+Percentage of deployments causing a failure in production (e.g., service impairment or unplanned outage) and that subsequently require remediation:
 
-`deployments` that caused `incidents` is defined as the closest deployment before the incident start. A deployment might be responsible for multiple incidents.
+```text
+number of deployments that caused incidents / total number of deployments
+```
 
-When we aggregate Change failure rate we use the average of a period of time.
+Pulse uses the average value over a period of time to display aggregate change failure rates.
 
 !!! note
-    We have opted to avoid requiring a relationship between `incidents` and `deployments` or `changes` to simplify the data model reported.
-
-| Performance level[^1] | Value  |
-| --------------------- | ------ |
-| Low                   | 46-60% |
-| Medium                | 0-15%  |
-| High                  | 0-15%  |
-| Elite                 | 0-15%  |
+    We decided to avoid requiring a relationship between **incidents** and **deployments** or **changes** to simplify the data model reported.
+    
+    As such, Pulse considers the **deployment** that caused an **incident** the closest deployment before the start of that **incident**. The same deployment might be responsible for multiple incidents.
+
+Pulse determines your performance level for this metric as follows:
+
+| Performance level[^1] | Change failure rate  |
+| --------------------- | -------------------- |
+| Elite                 | 0-15%                |
+| High                  | 0-15%                |
+| Medium                | 0-15%                |
+| Low                   | 46-60%               |
```

Comment:
I checked the State of DeOps 2019 report and indeed these are the values listed there. But this raises a few questions over our specific implementation:

* Does this mean that you'll get an "Elite" performance level no matter the change failure rate between 0 to 15%?
* What performance level will you get if you have a change failure rate between 16 and 45% or more than 60%?

:thinking:

Reply:
According to [our code](https://github.com/codacy/pulse/blob/master/modules/events/metrics-api/internal/persistence/queries/events_metrics.go#L243):
```suggestion
| Elite                 | 0-15%                |
| Medium                | 15-46%                |
| Low                   | 46-60%               |
```